.cicd/base-images.yml

```diff
@@ -22,6 +23,7 @@ steps:
     agents:
       queue: "automation-eks-eos-builder-fleet"
     timeout: 180
+    skip: ${SKIP_CENTOS_7}${SKIP_LINUX}
```

Comment:
The `SKIP_CENTOS_7` flag is inconsistent, albeit much better, than what we have in the [pipeline documentation](https://github.com/EOSIO/eos/blob/3950ef279b948cdaa83be3ab4e3b5355152395d8/.cicd/README.md#variables), which uses `SKIP_CENTOS_7_7`. Should these be consistent?